packages/strapi-parts/src/TextInput/TextInput.stories.mdx

```diff
@@ -11,11 +12,32 @@ import { Tooltip } from '../Tooltip';
 
 # TextInput
 
-This is the doc of the `TextInput` component
+TextInputs are inputs that allow users to add or edit text. They support several text formats including numbers.
```

Comment:
They don't include number. There's a NumberInput component for this :)

Reply:
You can write a number in a textInput right? 😂 Some textInputs don't allow special characters, numbers, or else so that's why I mentioned it @mfrachet

Reply:
oooh sorry, I didn't get it that way 🤦🏻

Reply:
Tt can be confusing. Should we mention that?

Reply:
+1. I'll add something to describe it better, and also, recommend the NumberInput for the case Marvin mentioned!